doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -306,6 +306,8 @@ Bluetooth samples
 
     * Aligned timers' configurations to the new nrfx API.
     * Extracted the DTM radio API from the transport layer.
+    * Added support for the radio fast ramp up feature.
+      This functionality is enabled by default.
```

Comment:
functionality -> feature?
Because the previous line talks about the radio fast ramp up feature.

Reply:
Can be feature and rump-up